flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/PartitionLoader.java

```diff
@@ -154,8 +166,82 @@ private void moveFiles(List<Path> srcDirs, Path destDir) throws Exception {
         }
     }
 
+    /**
+     * Reuse of PartitionCommitPolicy mechanisms. The default in Batch mode is metastore and
+     * success-file.
+     */
+    private void commitPartition(LinkedHashMap<String, String> partitionSpec, Path path)
+            throws Exception {
+        PartitionCommitPolicy.Context context = new CommitPolicyContextImpl(partitionSpec, path);
+        for (PartitionCommitPolicy policy : policies) {
+            if (policy instanceof MetastoreCommitPolicy) {
+                if (partitionSpec.isEmpty()) {
+                    // Non partition table skip commit meta data.
+                    continue;
+                }
+                ((MetastoreCommitPolicy) policy).setMetastore(metaStore);
+            }
+            policy.commit(context);
+        }
+    }
+
     @Override
     public void close() throws IOException {
         metaStore.close();
     }
+
+    /**
+     * Reuse the PartitionCommitPolicy in Hive batch sink. We only need partitionPath and
```

Comment:
The word `Hive` shouldn't appear in filesystem connector.

Reply:
The comment is a little of confused.  I think the comment isn't nesscessay since it's quite simple and the interface `PartitionCommitPolicy.Context` has done some explanation.